src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,397 @@
+updated: 2021-11-03
+category: background
+ingress:
+  This article introduces how payments work in Flex in general, and
+  describes the default Stripe payment gateway integration
+published: false
+---
+
+Flex is a full-fledged marketplace solution, complete with payment
+capabilities. In this article, you will learn about the Flex default
+payment integration, implemented with Stripe, as well as the
+alternatives in case you decide to not use Stripe on your marketplace.
+
+## Marketplace payment flow
+
+In a nutshell, a basic payment flow in a marketplace contains five
+significant steps:
+
+![Payment gateway logic](./payment-flow.png)
+
+### Step 1: Provider onboarding
+
+In this step, the provider connects their Flex account with the payment
+gateway. This is where they provide the bank details that eventually
+receive money from the customers. In addition, in this step, they
+provide the necessary information and documents for the identity
+verification and _Know Your Customer_
+[(KYC)](https://en.wikipedia.org/wiki/Know_your_customer) requirements.
+[These requirements vary](https://stripe.com/docs/connect/required-verification-information)
+depending on the user’s country of residence.
+
+### Step 2: Customer checkout
+
+Customer checkout happens when the customer initiates a transaction. At
+this stage, they also provide the payment information, such as their
+credit card number. Also, the payment will be made at this point. The
+payment gateway will preauthorize the money, i.e. reserve the money on
+the customer's credit card.
+
+### Step 3: Provider acceptance
+
+After the customer has checked out, the provider has the ability to
+either accept or reject the request. If the request is accepted, the
+payment will be captured, and the reserved money will be transferred
+from the customer's credit card to the payment gateway.
+
+This is a step that you can combine with the customer checkout. The flow
+where the “provider accept” happens instantly after customer checkout is
+called [instant booking](#instant-booking) flow.
+
+### Step 4: Customer refund
+
+Typically, the marketplace payment flow contains a delayed payment
+period. This is the time between when the money is captured from the
+customer's credit card and when it is transferred to the provider's bank
+account. The payout in marketplaces usually happens after the provider
+has successfully provided the agreed service.
+
+Customer refund usually happens during this delayed payment period.
+There are many reasons why a refund may be necessary. For example, the
+provider or customer may not be able to make it, or the provided service
+was not what was agreed.
+
+### Step 5: Provider payout
+
+If everything in the transaction went right and the customer received
+the agreed service, the money from the payment gateway will be
+eventually paid out to the provider.
+
+## Stripe default integration
+
+In the default Flex implementation, the process described above is
```

Comment:
Should we phrase this somehow like _In the default transaction process in Flex, the process described..._ to emphasize that by modifying the transaction process, the payment integration and flow can be changed? Or does _implementation_ also refer to FTW?

Reply:
Good point, clarified to `In the default Flex transaction process and FTW templates`